languages/go/exercises/concept/maps/.docs/instructions.md

```diff
@@ -13,34 +13,67 @@ A friend of you has an old wholesale store called **Gross Store**, the name came
 
 In order to use the measurement, you need to store the measurement in your program.
 
-## 2. Implement `AddItem`
+## 2. Add item to the customer bill
 
-This function will add item to the customer bill, to implement this, you'll need to:
+To implement this, you'll need to:
 
 - Check whether the given unit of measurement is correct
-- Add the item to the customer bill
+- Add the item to the customer bill, indexed by the item name. You probably also need a variable to represent the customer bill, you are expected to use `map`
 
-## 3. Implement `RemoveItem`
+```go
+ok := AddItem("carrot", "dozen")
+fmt.Println(ok)
+// Output: true or false
+```
 
-This function will remove item from the customer bill, to implement this, you'll need to:
+> Note that the returned value is type of `bool`
+
+## 3. Remove item from the customer bill
+
+To implement this, you'll need to:
 
 - Check whether the given item is in the bill
 - Check whether the given unit of measurement is correct
 - Check whether the new quantity is less than 0, is so return `false`
 - Check whether the new quantity is 0, is so return remove the item from the customer bill
 - Otherwise reduce the quantity of the item
 
-## 4. Implement `GetItem`
+```go
+ok := RemoveItem("carrot", "dozen")
+fmt.Println(ok)
+// Output: true or false
+```
+
+> Note that the returned value is type of `bool`
+
+## 4. Return the number of specific item that is in the customer bill
 
-This function will return the number of specific item that the customer has in his/her bill, to implement this, you'll need to:
+To implement this, you'll need to:
 
 - Check whether the given item is in the bill
 - Otherwise, return the quantity of the item
 
-## 5. Implement `Checkout`
+```go
+qty, ok := GetItem("carrot")
+fmt.Println(qty)
+// Output: 12
+fmt.Println(ok)
+// Output: true or false
+```
 
-This function supposed to do a payment to the customer bank, since you're developing a prototype, you'll just need to:
+> Note that the returned value are types of `int` and `bool`
+
+## 5. Simulate checkout operation
+
+This function supposed to do a payment to the customer bank, since you're developing a prototype, what you need to do is:
 
 - Reset the customer bill
 
+```go
+Checkout()
+// Output:
```

Comment:
As there is no output, feel free to omit this line:

```suggestion
```

You've already nicely explained this in the comment below